Networking/Networking/Mapper/MetaDataMapper.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+
+
+/// Mapper: MetaData List
+///
+struct MetaDataMapper: Mapper {
+
+    /// (Attempts) to convert an instance of Data into an array of MetaData Entities.
+    ///
+    func map(response: Data) throws -> [MetaData] {
+        let decoder = JSONDecoder()
+        let envelope = try decoder.decode(MetaDataEnvelope.self, from: response)
```

Comment:
Since [this](https://github.com/woocommerce/woocommerce-ios/pull/13929/files#diff-33032ebdb224cc4b33af7c7bc2270204221321aff5260e80e384c7ffb629ceceR31) is available as a REST request, should we update the `MetaDataMapper` to handle response with the "data" envelope by using the `hasDataEnvelope` method?

Reply:
👍 implemented here b1928b62145bce26461f688c6ff30279924c7bd2